### ISSUE TITLE
[BUG] tsed cli project with pnpm returns 404 on minimal hello-world example

### ISSUE THREAD

**Issue by dyerw** [TypedProject/ts-express-decorators#1679]
## Information

- **Version:** 6.95.2
- Packages: 

```
➜ cat package.json | grep tsed
    "@tsed/ajv": "^6.95.2",
    "@tsed/common": "^6.95.2",
    "@tsed/core": "^6.95.2",
    "@tsed/di": "^6.95.2",
    "@tsed/exceptions": "^6.95.2",
    "@tsed/json-mapper": "^6.95.2",
    "@tsed/platform-express": "^6.95.2",
    "@tsed/schema": "^6.95.2",
```

Using the tsed cli command to setup a project and choosing pnpm as the package manager results in a broken project. The routes are not correctly registered for some reason and attempting to hit them returns a 404.

## Example
These CLI options:
```
➜ tsed init example
? What is your project name example
? Choose the target platform: Express.js
? Choose the architecture for your project: Ts.ED
? Choose the convention file styling: Ts.ED
? Check the features needed for your project 
? Choose the package manager: PNPM
↓ Write RC files [SKIPPED]
✔ Install plugins
✔ Load plugins
✔ Install plugins dependencies
✔ Generate project files
```

(in my testing it is broken for Express and Koa)

then 
```
pnpm start
```

gives:
```
[INFO] 15:07:15 ts-node-dev ver. 1.1.8 (using ts-node ver. 9.1.1, typescript ver. 4.5.2)
Debugger listening on ws://127.0.0.1:9229/cb513dc7-2839-4f8b-a3ac-70e6f74e7c9e
For help, see: https://nodejs.org/en/docs/inspector
[2021-12-08T15:07:16.459] [DEBUG] [TSED] - Start server...
[2021-12-08T15:07:16.467] [INFO ] [TSED] - Injector created... +6ms
[2021-12-08T15:07:16.559] [INFO ] [TSED] - Providers loaded... +92ms
[2021-12-08T15:07:16.559] [INFO ] [TSED] - Build providers +0ms
[2021-12-08T15:07:16.563] [INFO ] [TSED] - Settings and injector loaded... +4ms
[2021-12-08T15:07:16.566] [INFO ] [TSED] - Load routes +3ms
[2021-12-08T15:07:16.568] [INFO ] [TSED] - Routes mounted... +2ms
[2021-12-08T15:07:16.569] [INFO ] [TSED] - 
┌────────┬──────────┬──────────────┐
│ Method │ Endpoint │ Class method │
└────────┴──────────┴──────────────┘
[2021-12-08T15:07:16.572] [INFO ] [TSED] - Listen server on http://0.0.0.0:8083
```

hitting `http://localhost/rest/hello-world`:

gives:

```
{"name":"NOT_FOUND","message":"Resource \"/rest/hello-world\" not found","status":404,"errors":[]}
```

## Acceptance criteria

- [ ] pnpm projects from the cli are able to start and return the hello world route without modification

**Comment by TalFaitlov** [comment]
Running into the same issue
In pnpm-based project controllers are not picked up
tsed version 6.11.3 + pnpm - WORKS

Let's fix it!

**Comment by Romakita** [comment]
Seems to be fixed with v6.102.5 :)

**Comment by Romakita** [comment]
The problem come from the peerDependencies which are resolved differently with PNPM :( 

Install the @tsed/logger on project level can solve the issue for previous version.
See you
Romain
